jcl/src/java.base/share/classes/openj9/internal/foreign/abi/UpcallMHMetaData.java

```diff
@@ -28,7 +28,7 @@
 import java.util.Optional;
 /*[ENDIF] JAVA_SPEC_VERSION >= 21 */
 
-/*[IF JAVA_SPEC_VERSION >= 20]*/
+/*[IF JAVA_SPEC_VERSION >= 21]*/
 /*[IF JAVA_SPEC_VERSION >= 21]*/
 import java.lang.foreign.AddressLayout;
 /*[ENDIF] JAVA_SPEC_VERSION >= 21 */
```

Comment:
Nested checks of `JAVA_SPEC_VERSION >= 21` and dead code in the nested `[ELSE]` block.

Reply:
Fixed.